packages/ui-react/src/components/Button/Button.tsx

```diff
@@ -56,6 +58,7 @@ const Button: React.FC<Props> = ({
       [styles.large]: size === 'large',
       [styles.small]: size === 'small',
       [styles.disabled]: disabled,
+      [styles.inHeader]: inHeader && variant === 'text',
```

Comment:
Try (re)using the `className` property instead of creating a new prop. Its not the Button-component responsibility t be aware of in which context it is used

Reply:
I agree. The button is already a little too complicated for its purpose. Simply passing a className should fix it?

Reply:
In the end the `activeClassname` prop was necessary, to apply the styling solely to an active button state